docs/src/api/class-browsercontext.md

```diff
@@ -1225,9 +1225,13 @@ Defaults to abort.
 
 ### option: BrowserContext.routeFromHAR.update
 * since: v1.23
-- `update` ?<boolean>
+- `update` ?<[HarUpdate]<"always"|"never"|"ifNotExists">>
```

Comment:
I don't know how I can handle an update of a property. That's why I left it here at v1.23 but maybe its not really a good idea because the new options are only available from new versions.